src/Microsoft.AspNetCore.Razor.Runtime/Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -138,33 +138,19 @@ public void Add(ITagHelper tagHelper)
         }
 
         /// <summary>
-        /// Tracks the minimized HTML attribute.
-        /// </summary>
-        /// <param name="name">The minimized HTML attribute name.</param>
-        public void AddMinimizedHtmlAttribute(string name)
```

Comment:
Note the baseline files made _zero_ use of this method: The code gen path using it was never entered.